### PR TITLE
Parse ScriptsPlayers properties

### PR DESCRIPTION
**Pull request by Skleni**
Adds conditional parsing of player properties for ScriptsPlayers.

### CODE REVIEW

File: src/OpenSage.Game/Data/Scb/ScriptPlayer.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿using System.IO;
+using OpenSage.Data.Map;
+using OpenSage.FileFormats;
+
+namespace OpenSage.Data.Scb
+{
+    public sealed class ScriptPlayer
```
Comment (Skleni) [inline src/OpenSage.Game/Data/Scb/ScriptPlayer.cs:7]: Or should this be Script*s*Player?

File: src/OpenSage.Game/Data/Scb/ScriptsPlayers.cs [hunk 2]
```diff
@@ -9,64 +8,67 @@ public sealed class ScriptsPlayers : Asset
...
+                ushort unknown = 0;
+                if (version < 2)
                 {
-                    throw new InvalidDataException();
+                    unknown = reader.ReadUInt16();
+                    if (unknown != 0)
+                    {
+                        throw new InvalidDataException();
+                    }
                 }
```
Comment (Skleni) [inline src/OpenSage.Game/Data/Scb/ScriptsPlayers.cs:36]: I couldn't find this value in Zero Hour maps/scripts.

File: src/OpenSage.Game/Data/Scb/ScriptPlayer.cs (hunk 1 above)
Comment (tgjones) [reply src/OpenSage.Game/Data/Scb/ScriptPlayer.cs:7]: I guess so, if you don’t mind renaming it, otherwise I’m happy to merge as is.

**Review by tgjones — APPROVED** [review]
Thank you!

**Pull request merged**